training/train.py

```diff
@@ -177,6 +176,7 @@ def train(
             y, y_pred = process_batch(batch, model)
 
             loss = criterion(y_pred, y)
+            avg_prob = calc_avgmax_attention(batch[-1], batch[1], y_pred[-1])
```

Comment:
Could we move this to line 197 and display in that message so that it is done every time we validate rather than every iteration